.github/workflows/continuous-integration.yml

```diff
@@ -15,10 +15,10 @@ jobs:
       - name: Execute linters and test suites
         run: ./docker/cibuild
```

Comment:
As `continuous-integration.yml` is launching `coverage` from within a docker container (`docker/cibuild`), the codecov-action silently fails, without finding the coverage file.

Reply:
issue is deferred via #190